modules/tensor_mechanics/tests/1D_axisymmetric/axisymm_plane_strain_finite.i

```diff
@@ -153,6 +146,7 @@
 
   [./strain]
     type = ComputeAxisymmetric1DFiniteStrain
+    out_of_plane_strain = strain_yy
```

Comment:
As I said earlier, I think it would be better to keep the current behavior where you don't need to specify this.  Then we wouldn't need to modify this input file.